documentation/asciidoc/computers/config_txt/common.adoc

```diff
@@ -0,0 +1,39 @@
+== Common Options
+
+=== Common Display Options
+
+==== `disable_overscan`
```

Comment:
I guess if this is being added here, it also ought to be removed from `video.adoc` ?
https://www.raspberrypi.com/documentation/computers/config_txt.html#disable_overscan

Reply:
@lurch can you do this one?

Reply:
Sorry, didn't see this comment until after you'd merged!
`hdmi_enable_4kp60` is also mentioned on both `common.adoc` and `video.adoc` ?

Reply:
Okay, I'll fix that directly on `develop` thanks @lurch!

Reply:
There appears to be "more" documentation on the video page, so perhaps it's worth just adding a cross-link? :shrug:

Reply:
Fixed up on `develop` directly and pushed to `master`.